README.md

```diff
@@ -66,23 +81,65 @@ app.get('/products',
   });
 ```
 
+### API Documentation:
+
+`auth()` accepts an asynchronous function receiving a token and returning a set of claims.
+
+`requiredScopes()` accepts either an string or an array of strings.
+
+`strategies.openid` accepts the following parameters:
+
+
+| Name                | Default                            | Description                                                          |
+|---------------------|------------------------------------|----------------------------------------------------------------------|
+| issuerBaseURL       | `env.ISSUER_BASE_URL`              | URL for the token issuer.                                            |
+| allowedAudiences    | `env.ALLOWED_AUDIENCES.split(',')` | Allowed audiences for the token.                                     |
+| clockTolerance      | `5`                                | Clock tolerance in seconds for token verification, aka leeway.       |
+| clientSecret        | `env.CLIENT_SECRET`                | Cleitn secret, required for tokens signed with symmetric algorithms. |
+
+## Contributing
+
+We appreciate feedback and contribution to this repo! Before you get started, please see the following:
+
+- [Auth0's general contribution guidelines](https://github.com/auth0/.github/blob/master/CONTRIBUTING.md)
+- [Auth0's code of conduct guidelines](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)
+
+Contributions can be made to this library through PRs to fix issues, improve documentation or add features. Please fork this repo, create a well-named branch, and submit a PR with a complete template filled out.
+
+Code changes in PRs should be accompanied by tests covering the changed or added functionality. Tests can be run for this library with:
```

Comment:
Maybe this should be part of a `DEVELOPING` file in the root

Reply:
At some point, maybe ... just wanted to get this information in there for now.